users/rmeli/global_config.h

```diff
@@ -0,0 +1,47 @@
+/*
+Copyright 2021-2022 Rocco Meli <@RMeli>
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#ifndef RMELI_GLOBAL_CONFIG
+#define RMELI_GLOBAL_CONFIG
```

Comment:
This file should be renamed to `config.h`, as that is automatically picked up. 

```suggestion
#pragma once
```